openshift/project.yaml

```diff
@@ -0,0 +1,3 @@
+project:
```

Comment:
What reads this file?

Reply:
This is read by the dockerfile generator